src/vivarium_census_prl_synth_pop/components/person_migration.py

```diff
@@ -24,16 +26,27 @@ class PersonMigration:
     The population at risk for all three move types is all simulants.
     """
 
-    def __repr__(self) -> str:
-        return "PersonMigration()"
-
     ##############
     # Properties #
     ##############
 
     @property
-    def name(self):
-        return "person_migration"
+    def columns_required(self) -> List[str]:
+        return [
+            "household_id",
+            "relationship_to_reference_person",
+            "previous_timestep_address_id",
+        ]
+
+    @property
+    def initialization_requires(self) -> Dict[str, List[str]]:
+        return {
+            "requires_columns": ["previous_timestep_address_id"],
```

Comment:
In the previous version it looks like this component creates the column `previous_timestep_address_id` but here you're saying it requires it in initialization.

Reply:
Yes this was a mistake.